src/calendar/view/CalendarView.ts

```diff
@@ -240,6 +240,7 @@ export class CalendarView extends BaseTopLevelView implements TopLevelView<Calen
 									},
 									selectedDate: this.viewModel.selectedDate(),
 									onDateSelected: (date, viewType) => {
+										this.viewModel.selectedDate(date)
```

Comment:
why do we need this one as well as URL? wouldn't setting the URL trigger it already?

Reply:
Sorry, that was not intended to be in the commit. I made a change here when I was testing and forgot to remove it.

That being said, we do this pattern for day as well. Maybe we should re-evaluate this?

https://github.com/tutao/tutanota/blob/227b7e440965de46a0cc0f766555b001cbde1442/src/calendar/view/CalendarView.ts#L214-L215

Reply:
hm yeah we should check if we need it at all or not

Reply:
Seems to not be needed.